.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- 
+    NOTE: Did you check the FAQs (https://one-dark.gitbook.io/jetbrains/help/faqs)
+    to see if your problem already has a work around?
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+1. Click on '....'
+1. Scroll down to '....'
+1. See error
+
+**Actual Behaviour**
+What is happening that you do not want to be happening
+
```

Comment:
I find that this section tends to just be a duplicated section from the describe the bug section.

```suggestion
```